pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
i do not have a good reason for using `POST` instead of `DELETE` beyond that's what we used for `filesystem delete`.  I can change this to use `DELETE` and then update the client also.